hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -302,25 +302,59 @@
   regwidth: "32",
   registers: {
     core: [
-      { name: "FLASH_DISABLE",
+      { name: "DIS",
         desc: "Disable flash functionality",
         swaccess: "rw1s",
```

Comment:
Should this now be a plain `rw` register?

Reply:
i think this is still okay. So in the future if you write it to all 1's, there will be no way to set it back to "disable".  This is meant to be somewhat one way. So really `rw1s` or `rw0c`, `rw1c` would all work.  Let me know if there's one you prefer.

Reply:
Ah I see. My slight preference would be `rw0c` since it requires one less instruction and so is slightly easier to do in assembly but feel free to leave as-is.

Reply:
o sounds good. i can change it to `rw0c`